eng/common/scripts/Prepare-Release.ps1

```diff
@@ -64,6 +64,55 @@ function Get-ReleaseDay($baseDate)
   return $baseDate;
 }
 
+function Check-ApiReviewStatus($packageName, $language)
```

Comment:
We should start thinking about creating a set of helpers for ApiView specific calls that we can import in the various places like this instead of having this directly in Prepare-Release